comp/swerve100/src/main/java/org/team100/frc2024/motion/drivetrain/ShooterUtil.java

```diff
@@ -7,25 +7,23 @@
 import org.team100.frc2024.FieldConstants;
 import org.team100.lib.motion.drivetrain.SwerveState;
 
-import edu.wpi.first.math.MathUtil;
 import edu.wpi.first.math.geometry.Translation2d;
 
-/** Add your docs here. */
 public class ShooterUtil {
 
-    public static Translation2d getOffsetTranslation(SwerveState state, double kScale){
-        Translation2d currentTranslation = state.pose().getTranslation();
+    public static Translation2d getOffsetTranslation(SwerveState state, double shooterVelocity){
```

Comment:
why is this change in here?

Reply:
Make a new method for this. One attempts to address shooting in the corner of the shooter and the other attempts to address shooting while moving